### ISSUE TITLE
Shaped Charges / Trip Mines cause WolfHud Crash in Beta

### ISSUE THREAD

**Issue by Anxarch** [Kamikaze94/WolfHUD#9]
Hello.

In the current Skills Beta, if you select Trip Mines, the game will crash if WolfHud is your current HUD.  I've removed every other mod except for WolfHud so it's one of the components.  It looks like something to do with teammate panels, but I'm not sure, as the only thing I've changed is my deployable.

```
Application has crashed: C++ exception
[string "lib/managers/hud/hudteammate.lua"]:745: attempt to index field '_player_panel' (a nil value)



SCRIPT STACK



-------------------------------

Callstack:

         payday2_win32_release  (???)     ???                                                 
         payday2_win32_release  (???)     ???                                                 
         payday2_win32_release  (???)     ???                                                 
         payday2_win32_release  (???)     ???                                                 
                         ntdll  (???)     RtlQueryPackageClaims                               


-------------------------------

Current thread: Main

-------------------------------
```

I believe this is probably due to Shaped Charges and Trip Mines being separate now and it's probably referencing an old value.  Changing my deployable away from Trip Mines fixes the problem.

Thanks.

**Comment by Kamikaze94** [comment]
Yeah, I got to know that yesterday, by a user on Lastbullet. 
It is caused by CustomHUD. I was told, that disabling CustomHUD in the Options works just fine...
I have a version of the new CustomHUD, found as TeammatePanels.lua on Sevens repo.
I'm currently modifying that version to my likeing and adding the changes I made to the currently used CustomHUD.
The new version has a far more complicated but also more stable "script layout". 
I don't know, if switching to that version will do the trick, or still need additional work.

Also thanks for reporting, and finding out, when/how its caused.
